client/modules/App/components/Login/Login.js

```diff
@@ -13,6 +13,17 @@ export class Login extends Component {
           <h1>
             Login
           </h1>
+          <form>
+            <label>
```

Comment:
things needed. 

- binding the form submission to `handleSubmit` function
- saving the value of the input for the password and email with `handleChange`